### PR TITLE
ledger: Adds bash completion

### PR DESCRIPTION
**Pull request by colindean**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Bash completion scripts have shipped for some time so let's add it in.

### CODE REVIEW

File: Formula/ledger.rb [hunk 1]
```diff
@@ -53,6 +53,7 @@ def install
     pkgshare.install "contrib"
     pkgshare.install "python/demo.py" if build.with? "python"
     elisp.install Dir["lisp/*.el", "lisp/*.elc"]
+    bash_completion.install pkgshare/"contrib/ledger-completion.bash"
```
Comment (fxcoudert) [inline Formula/ledger.rb]: Can you please report to upstream that these files really should be handled by `make install` itself?

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Review by ilovezfs — APPROVED** [review]

**Comment by dunn** [comment]
@brewtestbot please test this

File: Formula/ledger.rb (hunk 1 above)
Comment (ilovezfs) [reply Formula/ledger.rb]: Their bug tracker has been down for weeks http://bugs.ledger-cli.org/ and there's almost nothing with support for installing completions via `make install`, which is why I've approved this as-is.

**Comment by dunn** [comment]
@brewtestbot test this please

**Comment by dunn** [comment]
Merged as 15740f7f28b24d26cb8981c419e98749b2bc29d8, thanks!

**Pull request closed without merging**